.github/workflows/autopromote.yml

```diff
@@ -7,9 +7,9 @@ on:
       - development
 
 jobs:
-  update-pre-alpha:
+  update-alpha:
```

Comment:
Since we only have a single release stream now, I'm guessing the reason for the alpha is to provide a separate stream for testing out new Wynncraft versions on Wynncraft hero beta which introduce breaking changes. Am I correct?

If so, maybe we should name it "testing" instead of "alpha"? It feels a bit odd to "revert" to an alpha once we have done a series of proper releases.

Reply:
I would like it to keep the name as it fits into the conventional commit standard, even if it does not matter fully at the moment.